proxy/core/acceptor/acceptor.py

```diff
@@ -117,8 +134,15 @@ def run_once(self) -> None:
                         if mask & selectors.EVENT_READ:
                             if self.sock is not None:
                                 conn, addr = self.sock.accept()
-                                addr = None if addr == '' else addr
-                                self._work(conn, addr)
+                                logging.debug(
+                                    'Accepting new work#{0}'.format(conn.fileno()),
+                                )
+                                if self.flags.local_executor:
+                                    assert self._local
+                                    self._local.evq.put((conn, addr))
+                                else:
+                                    addr = None if addr == '' else addr
+                                    self._work(conn, addr)
```

Comment:
this could probably be
```suggestion
                                    self._work(conn, addr or None)
```